SU-HSDO/suhumsci
# ARCH-158 center button (more-link) links in views

# READY FOR REVIEW

# Summary
- Center button (more-link) links in views
- Reduced spacing above button (this paired with the reduced margin in https://github.com/SU-HSDO/suhumsci/pull/41 will get us where we want to be

# Needed By (Date)
- End of sprint (7/19)

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Ensure that the homepage featured views' buttons are centered

# Affected Projects or Products
- Archeaology
- suhumsci (theme)

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/ARCH-158
- BUGHERD-12
- https://github.com/SU-HSDO/suhumsci/pull/41

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
